src/features/game/expansion/Land.tsx

```diff
@@ -191,7 +191,7 @@ export const Land: React.FC = () => {
             style={{
               top: "50%",
               left: "50%",
-              zIndex: 100,
+              zIndex: 200,
```

Comment:
This prevents the `MapPlacement` elements from overriding the the drag and drop building getting placed. 

This is no necessarily desirable in all cases. A firepit should be draggable behind a fully grown sunflower for example.

![bakery](https://user-images.githubusercontent.com/41215134/184061219-062b892c-628e-4420-b9d3-e3cd41f6836b.png)